articles/tokens/access-token.md

```diff
@@ -6,15 +6,25 @@ toc: true
 
 ## Overview
 
-The Access Token, commonly referred to as `access_token` in code samples, is a credential that can be used by a client to access an API. The `access_token` should be used as a `Bearer` credential and transmitted in an HTTP `Authorization` header to the API. Auth0 uses access tokens to protect access to the Auth0 Management API.
+The Access Token, commonly referred to as `access_token` in code samples, is a credential that can be used by a client to access an API. The `access_token` can be any type of token (not necessarily a JSON Web Token) and is meant for the API. The purpose of the `access_token` is to inform the API that the bearer of this token has been authorized to access the API and perform specific actions (as specified by the `scope` that has been granted). The `access_token` should be used as a `Bearer` credential and transmitted in an HTTP `Authorization` header to the API. 
 
-Auth0 generates access tokens in [JSON Web Token (JWT)](/jwt) format, an industry standard. JWTs contain three parts: a header, a set of claims, and a signature:
-- The header contains metadata about the type of token and the cryptographic algorithms used to secure its contents.
-- The set of claims contains verifiable security statements such as the identity of the user and the permissions they are allowed.
-- The signature is used to validate that the token is trustworthy and has not been tampered with.
+## Access Tokens at Auth0
+
+Auth0 currently generates access tokens in two formats:
+
+* As opaque strings, when `/userinfo` is the audience (no custom API audience, no openid scope).
+* As a [JSON Web Token (JWT)](/jwt), an industry standard. Access tokens may be generated as JWTs when a custom API audience is specified with an `openid` scope. Both the client and the API will also need to be using the same signing algorithm (RS256/HS256) in order to get and use a properly formed JWT access token. 
+
+The client signing algorithm can be specified in the [Dashboard](${manage_url}) under the client's settings -> Advanced Settings -> Oauth. 
+
+![Token Signing Algorithm - Client](/media/articles/tokens/tokens-algorithm-client.png)
+
+When setting up an API in the [Dashboard](${manage_url}/#/apis), the signing algorithm can also be specified.
+
+![Token Signing Algorithm - API](/media/articles/tokens/tokens-algorithm-api.png)
 
 ::: note
-The [Auth0 Management API v1](/api/management/v1) (which has been deprecated) uses an opaque token format in which claims are referenced in a separate database, rather than directly in the token.
+While Auth0 currently uses JWTs for many access tokens, the important thing to remember is that the client should not be depending on the access token to be any specific format, but instead should treat the access token as opaque.
```

Comment:
I would remove this `While Auth0 currently uses JWTs for many access tokens`

Reply:
And maybe use the `warning` UI element since this is something we want to put emphasis on